src/internal/utils/__tests__/node-belongs.test.ts

```diff
@@ -34,10 +34,18 @@ describe('nodeBelongs', () => {
     const div = document.createElement('div');
     div.innerHTML = `
       <div id="container1">
+        <div id="portal"></div>
+      </div>
+      <div data-awsui-referrer-id="portal">
         <div id="node"></div>
       </div>
-      <div data-awsui-referrer-id="node"></div>
     `;
-    expect(nodeBelongs(div.querySelector('#container1'), div.querySelector('#node') as Node)).toBe(true);
+    document.documentElement.appendChild(div);
+
+    try {
```

Comment:
Using try in tests is an instant red flag.

1. If you the exception is expected here, it should be `expect(() => ...).toThrow()`
2. If unexpected and you want to do a cleanup, use beforeEach/afterEach to render a utility diff

Reply:
Thanks @connorlanigan, resolving this